roofit/roofitcore/test/TestStatistics/testLikelihoodGradientJob.cpp

```diff
@@ -100,12 +101,17 @@ TEST_P(LikelihoodGradientJob, Gaussian1D)
    *values = *savedValues;
 
    RooFit::MultiProcess::Config::setDefaultNWorkers(NWorkers);
-   auto likelihood = std::make_shared<RooFit::TestStatistics::RooUnbinnedL>(pdf, data);
-   RooMinimizer m1(likelihood, RooFit::TestStatistics::LikelihoodMode::serial,
-                   RooFit::TestStatistics::LikelihoodGradientMode::multiprocess);
+   auto unbinned_l = std::make_shared<RooFit::TestStatistics::RooUnbinnedL>(pdf, data);
+   RooFit::TestStatistics::RooRealL likelihood("likelihood", "likelihood", unbinned_l);
 
-   m1.setStrategy(0);
-   m1.setPrintLevel(-1);
+   // Convert to RooRealL to enter into minimizer
+   RooMinimizer::Config cfg;
+   cfg.parallelLikelihood = false;
+   cfg.parallelGradient = true;
+   RooMinimizer m1(likelihood, cfg);
+
+   RooMinimizer::setStrategy(0);
```

Comment:
If I do not here do this again the test fails. It fails with some small deviations with respect to the reference, so I am assuming the strategy actually does get set to a different value if I do not do this here again.
 
This is probably not the behaviour that the user expects from these static functions.